docs/sbpy/data/dataclass.rst

```diff
@@ -576,7 +576,15 @@ Writing object data to a file
 `~sbpy.data.DataClass` objects can be written to files using
 `~sbpy.data.DataClass.to_file`:
 
-    >>> obs.to_file('observations.dat')
+    .. testsetup:: obs.to_file
+        >>> import os
+        >>> assert not os.path.exists('observations.dat')
+
+    .. doctest:: obs.to_file
+        >>> obs.to_file('observations.dat')
+
+    .. testcleanup:: obs.to_file
+        >>> os.unlink('observations.dat')
```

Comment:
Nice solution. I would have thought you need to import os under testcleanup but apparently it is not necessary since all the checks have passed.

Reply:
I think that has something to do with the block name following the directives.  Here, I called the block obs.to_file.

Reply:
Hmmm, I guess the groups are only used inside of sphinx.